java/src/main/java/io/opentracing/contrib/ClientTracingInterceptor.java

```diff
@@ -299,7 +299,7 @@ public ClientTracingInterceptor build() {
         }
     }
 
-    enum ClientRequestAttribute {
+    public enum ClientRequestAttribute {
```

Comment:
This change is for https://github.com/grpc-ecosystem/grpc-opentracing/issues/17 and https://github.com/grpc-ecosystem/grpc-opentracing/issues/18